summary.txt

```diff
@@ -1 +1 @@
-This version fixes the problem of obelisks vanishing into thin air when trying to put items inside.
\ No newline at end of file
+This version implements most of the WIP spellparts, and includes various minor changes and fixes.
```

Comment:
This should be the summary that is getting sent as the update notification on discord (maybe remove the `This version` part)